.github/workflows/checks.yaml

```diff
@@ -254,6 +254,62 @@ jobs:
             RUST_BACKTRACE=1 cargo fuzz run --no-default-features --features="$features" $target -- -max_total_time=10
           done
 
+  fuzz-code-coverage:
+    name: Fuzz with code coverage 
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - fuzz_target: uncompress2
+            corpus: "gzip-files/compressed"
+            features: '--no-default-features --features="disable-checksum"'
+            flags: fuzz-decompress
+          - fuzz_target: compress
+            corpus: ""
+            features: ''
+            flags: fuzz-compress
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          persist-credentials: false
+      - name: Install nightly toolchain
+        uses: dtolnay/rust-toolchain@be73d7920c329f220ce78e0234b8f96b7ae60248
+        with:
+          toolchain: nightly
+          components: llvm-tools-preview
+      - name: Install cargo fuzz & rustfilt
+        uses: taiki-e/install-action@56ab7930c591507f833cbaed864d201386d518a8
+        with:
+          tool: cargo-fuzz,rustfilt
+      - name: Download custom decompression corpus
+        if: ${{ contains(matrix.corpus, 'gzip-files') }}
+        run: |
+          ~/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin/llvm-cov show --help
```

Comment:
You could use `$(rustc --print sysroot)` instead of hard-coding `~/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu`.